pom.xml

```diff
@@ -1080,6 +1080,7 @@ under the License.
           <artifactId>maven-release-plugin</artifactId>
           <configuration>
             <autoVersionSubmodules>true</autoVersionSubmodules>
+            <releaseProfiles>apache-release,reporting</releaseProfiles>
```

Comment:
site is not build during release - so this has no effect here

Reply:
why not? Don't we rely on the default from https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html#goals

Reply:
we use only `deploy` goal 
https://github.com/apache/maven-apache-parent/blob/master/pom.xml#L248-L258

Reply:
Any idea why? IMHO building and deploying site also is very useful (at least for Msven Plugins)

Reply:
I created https://issues.apache.org/jira/browse/MPOM-359 for that.

Reply:
By the way I am for removing reporting  profile at all - all site reports should be build  always without special profile.
But should be discussed on dev list - maybe exist a rationale reason why we have special profile.